ONSdigital/design-system
# Added guidance on writing URLs and updated DS URLs to follow these rules

### What is the context of this PR?
Added guidance on writing URLs and updated DS URLs to follow these rules

### How to review
- Check documentation
- Check for broken links caused by folder name changes
